ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -0,0 +1,332 @@
+        logger.info("Create UFS model readable atm increment file from UFS-DA atm increment")
+        self.jedi2fv3inc()
+
+    def clean(self):
+        super().clean()
+
+    @logit(logger)
+    def get_bkg_dict(self, task_config: Dict[str, Any]) -> Dict[str, List[str]]:
+        """Compile a dictionary of model background files to copy
+
+        This method constructs a dictionary of FV3 RESTART files (coupler, core, tracer)
+        that are needed for global atmens DA and returns said dictionary for use by the FileHandler class.
+
+        Parameters
+        ----------
+        task_config: Dict
+            a dictionary containing all of the configuration needed for the task
+
+        Returns
+        ----------
+        bkg_dict: Dict
+            a dictionary containing the list of model background files to copy for FileHandler
+        """
+        # NOTE for now this is FV3 RESTART files and just assumed to be fh006
+
+        bkgdir = [
+            os.path.join(task_config.DATA, 'bkg'),
+        ]
+        FileHandler({'mkdir': bkgdir}).sync()
+
+        # loop over ensemble members
+        bkglist = []
+        for imem in range(1, task_config.NMEM_ENKF + 1):
+            memchar = f"mem{imem:03d}"
+
+            # make run directory for member restart files
+            bkgdir = [
+                os.path.join(task_config.DATA, 'bkg', memchar)
+            ]
+            FileHandler({'mkdir': bkgdir}).sync()
+
+            # get FV3 RESTART files, this will be a lot simpler when using history files
+            rst_dir = os.path.join(task_config.comin_ges_atmens, memchar, 'atmos/RESTART')
+            run_dir = os.path.join(task_config.DATA, 'bkg', memchar)
+
+            # atmens DA needs coupler
+            basename = f'{to_fv3time(task_config.current_cycle)}.coupler.res'
+            bkglist.append([os.path.join(rst_dir, basename), os.path.join(task_config.DATA, 'bkg', memchar, basename)])
+            # atmens DA needs core, srf_wnd, tracer, phy_data, sfc_data
+            for ftype in ['core', 'srf_wnd', 'tracer']:
+                template = f'{to_fv3time(self.task_config.current_cycle)}.fv_{ftype}.res.tile{{tilenum}}.nc'
+                for itile in range(1, task_config.ntiles + 1):
+                    basename = template.format(tilenum=itile)
+                    bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+            for ftype in ['phy_data', 'sfc_data']:
+                template = f'{to_fv3time(self.task_config.current_cycle)}.{ftype}.tile{{tilenum}}.nc'
+                for itile in range(1, task_config.ntiles + 1):
+                    basename = template.format(tilenum=itile)
+                    bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        bkg_dict = {
+            'copy': bkglist,
+        }
+
+        return bkg_dict
+
+    @logit(logger)
+    def jedi2fv3inc(self: Analysis) -> None:
```

Comment:
Now that we have a repetition of this method here and in `atm_analysis.py`, we need to not repeat.